UnitsNet.Tests/CustomQuantities/HowMuch.cs

```diff
@@ -26,7 +26,7 @@ public HowMuch(double value, HowMuchUnit unit)
         public QuantityType Type => QuantityType.Undefined;
         public BaseDimensions Dimensions => BaseDimensions.Dimensionless;
 
-        public QuantityInfo QuantityInfo => new QuantityInfo(Type,
+        public QuantityInfo QuantityInfo => new QuantityInfo("HowMuch",
```

Comment:
This is a good improvement on its own, third party quantities get an actual name instead of Undefined quantity type.

Reply:
👍